invenio_rdm_records/services/community_inclusion/service.py

```diff
@@ -64,7 +66,23 @@ def include(self, identity, community, request, uow):
 
         if can_include_directly:
             request_item = current_requests_service.execute_action(
-                identity, request.id, "accept", data=None, uow=uow
+                system_identity, request.id, "accept", data=None, uow=uow
+            )
+
+            data = {
+                "payload": {
+                    "content": "This draft review request was automatically created and accepted because "
+                    "the record was published directly to community, bypassing the review "
+                    "process."
```

Comment:
Shall we make it more simple? What do you think of this?

`This request has been automatically accepted, as the uploader can submit directly without review.`